Add function for update x to near neighborhood.

seigodev opened the pull request:
座標決定機能の追加  #15
ノードのx座標をターゲットもしくはソースに近づくように更新する関数を追加しました。

seigodev committed:
Add function for update x to near neighborhood.

ノードのx座標をターゲットもしくはソースに近づくように更新する関数を追加しました。

aabaa commented on demo/create_graph.py:
```diff
@@ -493,6 +493,27 @@ def calc_edge_length_sum(all_nodes):
 """
 
 
+def update_node_x2near_the_neighborhood(all_nodes, downward):
```
関数の意味がわからないので，名前を変更してください．
むやみにtoの代わりに2を使わないこと．dictを生成するときのみにするように．
文脈から判断して，update_all_level_nodes()等のほうが良いです．
全般的に関数名がちぐはぐになってきているので一通り見直してください．
updateよりもmoveという動詞のほうがあっていると思います．

seigodev committed:
Rename function.

update_node_x2near_the_neighborhood() -> move_node_closer_to_connected_nodes()
関数名を書き直しました。
